### PR TITLE
Update UUID module for Chapel v2.0

### PR DESCRIPTION
**Pull request by astatide**
This PR adds a new version of the UUID module, which has been updated to work properly with Chapel 2.0.  It also updates to the new repo location on Codeberg.

### CODE REVIEW

**Comment by arezaii** [comment]
It looks like we (mason) need to update how we validate license names since SPDX archived their repo and changed where they publish the list.

File: Bricks/UUID/1.0.3.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[brick]
+name = "UUID"
+version = "1.0.3"
+chplVersion = "2.0.0"
+authors = "Audrey MP"
+type = "module"
+source = "https://codeberg.org/Astatide/chplUUID"
+license = "AGPL-3.0-or-later
```
Comment (arezaii) [inline Bricks/UUID/1.0.3.toml]: it looks like the license line is missing a closing `"`

Comment (astatide) [reply Bricks/UUID/1.0.3.toml]: Ah, good catch!  Thank you.

**Comment by arezaii** [comment]
I recently merged https://github.com/chapel-lang/chapel/pull/25062, which should fix the issue with the license names once you run `mason publish --refresh-licenses`

**Comment by astatide** [comment]
Aha, yay, looks like it works now.  Thank you!

**Review by arezaii — APPROVED** [review]

**Comment by astatide** [comment]
Oh, I forgot this was still open; I can't actually merge it on my own.  It looks like everything is ready to go, though, so is there anything else required for merging?

**Comment by lydia-duncan** [comment]
Ahmad's pretty busy with release stuff at the moment, but I'll set a reminder to either ask him and merge it myself or have him merge it when we're past the release (though all of that may be impacted by upcoming end of year vacation time)

**Comment by lydia-duncan** [comment]
Thanks for reminding us, though, we really appreciate it :)

**Comment by astatide** [comment]
No problem and no rush!  I was setting up a project with mason and testing how to actually add dependencies, then realized I pulled in the old version and couldn't remember if this is the module I had updated or not!

**Pull request merged**